### ISSUE TITLE
GPT 3.5 (Web) no longer available?

### ISSUE THREAD

**Issue by gitdine** [josStorer/chatGPTBox#721]
I noticed that with the recent update it now defaults to GPT 4 instead of 3.5, which is a lot slower and effects the usage cap.

Just wondering if it's possible to force GPT 3.5 when using the web interface? 

Not a huge problem, but I miss the speed of this extension with  GPT 3.5

**Comment by Mohamed3nan** [comment]
If you're using the extension, you can choose your default mode
the first one "ChatGPT (Web)" is the old GPT3.5

![image](https://github.com/josStorer/chatGPTBox/assets/19733801/4ef427e1-a969-474a-9d11-764a54efb3aa)
![image](https://github.com/josStorer/chatGPTBox/assets/19733801/a37d9e61-ff56-4b8c-99c6-ae2e3038c11b)

**Comment by gitdine** [comment]
When I ask it, it says it's GPT-4.
![image](https://github.com/josStorer/chatGPTBox/assets/55441362/6e017648-c5b2-429f-aff1-8f35fc746a77)

**Comment by Mohamed3nan** [comment]
yes it's because of OpenAI system prompt I guess they want to it act like GPT-4 :D

but If you go to the chat history you will notice it's 3.5 mode

![image](https://github.com/josStorer/chatGPTBox/assets/19733801/01aa6e9a-1bc6-410c-b1ac-e0075200bf01)
